src/lib/components/Search/index.js

```diff
@@ -223,8 +224,19 @@ class Search extends BaseElement {
     // This is intentional because focus needs to stay in the input field.
     // When the user is pressing arrow keys, we use a virtual cursor and
     // aria-activedescendant to indicate the active anchor.
-    return this.hits.map(
-      (hit, idx) => html`
+    return this.hits.map((hit, idx) => {
+      if (!hit._highlightResult.title || !hit._highlightResult.title.value) {
+        return html``;
+      }
+
+      let title = hit._highlightResult.title.value;
+      // Escape any html entities in the title except for <strong> tags.
+      // Algolia sends back <strong> tags in the title which help highlight
+      // the characters that match what the user has typed.
+      title = allowHtml(escapeHtml(title), 'strong');
+      // Strip backticks as they look a bit ugly in the results.
+      title = title.replace(/`/g, '');
```

Comment:
Not saying it's necessary, but you could use the `strip` filter for this => https://github.com/GoogleChrome/web.dev/blob/76eff20f910842979fcc56fceac2dd3af896eece/src/site/_filters/strip.js

Reply:
oh cool i'll do that!

Reply:
ran into weird issues trying to get strip (which is CJS) to import correctly into this file so I'm just going to leave the little regex.